pkg/virt-api/webhooks/validating-webhook/admitters/vmrestore-admitter_test.go

```diff
@@ -67,7 +67,7 @@ var _ = Describe("Validating VirtualMachineRestore Admitter", func() {
 		},
 	}
 
-	config, configMapInformer, _, _ := testutils.NewFakeClusterConfig(&corev1.ConfigMap{})
+	config, _, kvInformer := testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{})
```

Comment:
Nit: I guess we can drop the `KVConfig` now that we only have one function. `testutils.NewFakeClusterConfig()` would be sufficient now.